module1/src/com/epam/cycles/HomeTask_1_3_2.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.cycles;
+
+public class HomeTask_1_3_2 {
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 10;
+        int h = 1;
+        int y ;
+        for(int x = a; x <=  b; x += h) {
+            if (x > 2) {
```

Comment:
move 2 to constant, e.g. private static final double CONDITION_PARAMETER = 2.0;
```suggestion
            if (x > CONDITION_PARAMETER) {
```